src/github3/apps.py

```diff
@@ -171,26 +171,3 @@ def create_token(private_key_pem, app_id, expire_in=TEN_MINUTES_AS_SECONDS):
     )
     token.make_signed_token(key)
     return token.serialize()
-
-
-def create_jwt_headers(
```

Comment:
This is a publicly named function in a publicly named module. We can't just get rid of it

Reply:
Restored